heartbeat/pgsql

```diff
@@ -1479,7 +1479,7 @@ set_async_mode_all() {
 }
 
 set_async_mode() {
-    cat $REP_MODE_CONF |  grep -q -e "[,' ]$1[,' ]"
+    cat $REP_MODE_CONF |  grep -q -e "\"$1\""
```

Comment:
As reported in the postgresql [documentation](https://www.postgresql.org/docs/9.6/static/runtime-config-replication.html): 
> Each standby_name should have the form of a valid SQL identifier, unless it is *. 

and for an [identifier](https://www.postgresql.org/docs/9.1/static/sql-syntax-lexical.html):
> Quoted identifiers can contain any character, except the character with code zero. (To include a double quote, write two double quotes.) This allows constructing table or column names that would otherwise not be possible, such as ones containing spaces or ampersands.

With your suggestion you will match something like "don't match this" searching for "don't" , "match" or "this"
In my opinion the right backwards compatible change should be:
`grep -E "(\"$1\")|([,' ]$1[,' ])"`